t/mojolicious/log_lite_app.t

```diff
@@ -7,6 +7,9 @@ use Mojolicious::Lite;
 use Mojo::Log;
 use Test::Mojo;
 
+# Test::Mojo sets MOJO_LOG_LEVEL to fatal
+local $ENV{MOJO_LOG_LEVEL} = 'debug';
```

Comment:
Needing a workaround for the test that's supposed to actually cover logging seems like a very bad sign for this PR.